lib/src/matomo.dart

```diff
@@ -360,44 +366,43 @@ class MatomoTracker {
   /// Register an event with [eventName] as the event's name and [widgetName] as
   /// the event's action.
   ///
-  /// - `widgetName`: Equivalent to the event action, here used to identify the
-  /// screen with a proper name. This corresponds with `action_name`.
+  /// - `actionName`: Equivalent to the event action, here used to identify the
+  /// screen with a proper name.
   ///
-  /// - `eventName`: The name of the event. This corresponds with `e_n`.
+  /// - `eventInfo`: Additional data for the event send.
   ///
-  /// - `currentScreenId`: A 6 character unique ID that identifies which actions
-  /// were performed on a specific page view. If `null`, a random id will be
-  /// generated. This corresponds with `pv_id`.
+  /// - `pvId`: A 6 character unique ID that identifies which actions
+  /// were performed on a specific page view. If `null`, a random ID will be
+  /// generated.
   ///
   /// - `path`: A string that identifies the path of the screen. If not
-  /// `null`, it will be combined to [contentBase] to create a URL. This combination
-  /// corresponds with `url`.
+  /// `null`, it will be combined to [contentBase] to create a URL. This
+  /// combination corresponds with `url`.
   ///
   /// For remarks on [dimensions] see [trackDimensions].
   void trackScreenWithName({
-    required String widgetName,
-    required String eventName,
-    String? currentScreenId,
+    required EventInfo eventInfo,
```

Comment:
I think this is conceptually wrong here. `trackScreenWithName` should track a page view, not an event. Therefor, it should not take an `eventInfo` as parameter. With the current code what will happen is that the tracker sends a request with the parameters `e_c`, `e_a` and `e_n` to the server where it should send `action_name` instead.

Reply:
It would still send an `action_name` in the request thought but I understand your point. I'll find a better way to implement this API.